### PR TITLE
add option to compile with/without webengine

### PR DESCRIPTION
**Pull request by lasconic**

### CODE REVIEW

**Comment by lasconic** [comment]
Need more tests on Linux and Windows.

File: mscore/startcenter.cpp [hunk 1]
```diff
@@ -150,7 +150,7 @@ void Startcenter::readSettings()
       MuseScore::restoreGeometry(this);
       }
 
-#if 0
+#ifdef USE_WEBENGINE
```
Comment (Jojo-Schmitz) [inline mscore/startcenter.cpp]: ifdef here...

File: mscore/startcenter.h [hunk 2]
```diff
@@ -18,7 +18,7 @@
 
 namespace Ms {
 
-#if 0
+#if USE_WEBENGINE
```
Comment (Jojo-Schmitz) [inline mscore/startcenter.h]: ... but if here

**Comment by Jojo-Schmitz** [comment]
Works (i.e. compiles/builds without WebEngine) on Windows

File: mscore/startcenter.h (hunk 2 above)
Comment (Jojo-Schmitz) [inline mscore/startcenter.h]: For other similar ones (USE_ALSE, HAS_MIDI, USE_PULSEAUDIO, USE_LAME, ...) we use `#ifdef`. It does work with `#if' too though

Comment (Jojo-Schmitz) [inline mscore/startcenter.h]: But then again see http://stackoverflow.com/questions/135069/ifdef-vs-if-which-is-better-safer-as-a-method-for-enabling-disabling-compila, several good reasons to be using `#if`

**Comment by lasconic** [comment]
It needs more work. Especially for linking. I will look again another time. We need a solution for the start center anyway.

**Pull request closed without merging**